release-notes/2024-05/README.md

```diff
@@ -0,0 +1,207 @@
+
+# SystemLink Enterprise release 2024-05 Release Notes
+
+The 2024-05 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- `serviceregistry 0.14.19`
```

Comment:
I think it would make more sense to document permission changes for the changed service, not for the service registry, which just hosts the UI configuration.

Reply:
Since I cannot find the users for Gopal Sabarinathan and Kaviarasu Selvarasu, who have added these documentation notes, I did start a Teams group chat with them, + JD, + Surya, + Usha.

Reply:
@JD-Robertson , have replaced `serviceregistry 0.14.19` with `workorder 0.4.35`, per input from Gopal Sabarinathan in Teams